### ISSUE TITLE
Example Content: some content types are not listed in the main navigation (excluded from nav maybe?)

### ISSUE THREAD

**Issue by tisto** [kitconcept/volto-light-theme#305]
https://light-theme.kitconcept.io/content-types/image is not listed in the fat menu

**Comment by ichim-david** [comment]
@tisto Image has no workflow set and it doesn't show up in the navigation endpoint

**Comment by iRohitSingh** [comment]
@tisto I checked the content type in light theme. Link, file and image doesn't show up in navigation. Even I checked the network request exclude from nav is false on them. Maybe somehow backend doing filtering.

**Comment by iRohitSingh** [comment]
@davisagli can you please look and give me pointer where I should look into backend to debug this.

**Comment by davisagli** [comment]
@iRohitSingh The types that can show up in navigation are controlled by the plone.displayed_types registry setting: https://github.com/kitconcept/dlr-internet/blob/main/backend/src/dlr.internet/src/dlr/internet/profiles/default/registry/registry.xml#L13

But @tisto I don't think we want to enable showing these types in navigation just to include the example content. In general we don't want them in navigation.

**Comment by tisto** [comment]
Right, I forgot about those things. However, sometimes there are exceptions to this rule. For instance when you want to build a navigation structure that is not 1:1 the same as your content structure. In that case you might want to show a link in the navigation, even if the Link type (by default) is excluded from the navigation.

This is the reason why for all our client projects, we add the "exclude_from_navigation" checkbox to all content types and just set this to True for content types like links and images. I think this is a pattern that Plone core should adopt at some point. Therefore I guess this is something that we should either add to kitconcept.volto or plone.volto. cc @sneridagh

**Comment by davisagli** [comment]
@tisto There is an exclude_from_nav behavior in plone.app.dexterity, and it even has an existing adapter to provide custom logic for whether it should be checked by default or not: https://github.com/plone/plone.app.dexterity/blob/master/plone/app/dexterity/behaviors/exclfromnav.py

But, the default implementation always sets it to False. I think it would make more sense for the default implementation to return a value based on a registry setting. Then it's not necessary to write code to change it.

**Comment by sneridagh** [comment]
@tisto should we address this in the core Plone side then?
